jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java

```diff
@@ -426,9 +424,11 @@ public final long getProcessVirtualMemorySize() {
 	 */
 	@Override
 	public final double getSystemCpuLoad() {
-		return cpuUtilizationHelper.getSystemCpuLoad();
+		return this.getSystemCpuLoadImpl();
 	}
 
+	private native double getSystemCpuLoadImpl();
```

Comment:
This method can be `static`.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/e200fcd0089727bb8c11ffa2d51bc48880865951..a15fd0a5f1583c638085002e198a4ef9732245cf).

Reply:
The reason this was not done was to remain consistent with the other APIs within this same class. For example, by the same logic the following APIs could be made `static` as well:

https://github.com/eclipse/openj9/blob/0ae4f22e2d7769fb2796382ef2b33bcacb749ff2/jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java#L338

https://github.com/eclipse/openj9/blob/0ae4f22e2d7769fb2796382ef2b33bcacb749ff2/jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java#L353

https://github.com/eclipse/openj9/blob/0ae4f22e2d7769fb2796382ef2b33bcacb749ff2/jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java#L391

https://github.com/eclipse/openj9/blob/0ae4f22e2d7769fb2796382ef2b33bcacb749ff2/jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedOperatingSystemMXBeanImpl.java#L422

etc. (there are dozens more). As such making this new API `static` would be inconsistent with the others. Would it be reasonable to not deviate from these other similar APIs, and instead open up an issue to bulk-change them all to be `static`?

Reply:
I'm fine with following the pattern and opening an issue to clean up later (unless a static native is more expensive than a non-static one).

Reply:
I've opened up #8647 to track this work. Thanks for identifying the opportunity.